doc/developer/design/20250226_postgres_style_explain.md

```diff
@@ -0,0 +1,338 @@
+  ->  GroupAggregate  (cost=20.74..20.77 rows=1 width=44)
+        Group Key: lineitem.l_orderkey, orders.o_orderdate, orders.o_shippriority
+        ->  Sort  (cost=20.74..20.74 rows=1 width=48)
+              Sort Key: lineitem.l_orderkey, orders.o_orderdate, orders.o_shippriority
+              ->  Nested Loop  (cost=0.29..20.73 rows=1 width=48)
+                    ->  Nested Loop  (cost=0.14..19.93 rows=1 width=12)
+                          ->  Seq Scan on customer  (cost=0.00..11.75 rows=1 width=4)
+                                Filter: (c_mktsegment = 'BUILDING'::bpchar)
+                          ->  Index Scan using fk_orders_custkey on orders  (cost=0.14..8.16 rows=1 width=16)
+                                Index Cond: (o_custkey = customer.c_custkey)
+                                Filter: (o_orderdate < '1995-03-15'::date)
+                    ->  Index Scan using fk_lineitem_orderkey on lineitem  (cost=0.14..0.79 rows=1 width=40)
+                          Index Cond: (l_orderkey = orders.o_orderkey)
+                          Filter: (l_shipdate > '1995-03-15'::date)
+(16 rows)
+```
+
+Materialize `EXPLAIN`:
+
+```
+Finish order_by=[#1{sum} desc nulls_first, #2{o_orderdate} asc nulls_last] output=[#0..=#3]
+  Project (#0{o_orderkey}, #3{sum}, #1{o_orderdate}, #2{o_shippriority}) (columns=4)
+    Reduce group_by=[#0{o_orderkey}..=#2{o_shippriority}] aggregates=[sum((#3{l_extendedprice} * (1 - #4{l_discount})))] (columns=4)
+      Project (#8{o_orderkey}, #12{o_orderdate}, #15{o_shippriority}, #22{l_extendedprice}, #23{l_discount}) (columns=5)
+        Filter (#6{c_mktsegment} = "BUILDING") AND (#12{o_orderdate} < 1995-03-15) AND (#27{l_shipdate} > 1995-03-15) (columns=33)
+          Join on=(#0{c_custkey} = #9{o_custkey} AND #8{o_orderkey} = #17{l_orderkey}) type=delta (columns=33)
+            implementation
+              %0:customer » %1:orders[#1]KAif » %2:lineitem[#0]KAif
+              %1:orders » %0:customer[#0]KAef » %2:lineitem[#0]KAif
+              %2:lineitem » %1:orders[#0]KAif » %0:customer[#0]KAef
+            ArrangeBy keys=[[#0{c_custkey}]] (columns=8)
+              ReadIndex on=customer pk_customer_custkey=[delta join 1st input (full scan)] (columns=8)
+            ArrangeBy keys=[[#0{o_orderkey}], [#1{o_custkey}]] (columns=9)
+              ReadIndex on=orders pk_orders_orderkey=[delta join lookup] fk_orders_custkey=[delta join lookup] (columns=9)
+            ArrangeBy keys=[[#0{l_orderkey}]] (columns=16)
+              ReadIndex on=lineitem fk_lineitem_orderkey=[delta join lookup] (columns=16)
+
+Used Indexes:
+  - materialize.public.pk_customer_custkey (delta join 1st input (full scan))
+  - materialize.public.pk_orders_orderkey (delta join lookup)
+  - materialize.public.fk_orders_custkey (delta join lookup)
+  - materialize.public.fk_lineitem_orderkey (delta join lookup)
+
+Target cluster: quickstart
+```
+
+New Materialize `EXPLAIN`:
+
+```
+Finish
+  Order by: sum desc nulls_first, o_orderdate
```

Comment:
We could leave off `nulls_first`/`nulls_last` if they are at their default setting. (Which I think is `nulls_first` for `desc`, and `nulls_last` for `asc`.